nf_requirements/v1alpha1/upfdeployment_types.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2022 Authors of Project Nephio.
```

Comment:
2023 The Nephio Authors

I guess we can standardize on that one? @vjayaramrh @henderiw @gvbalaji @aravind254 @radoslawc

Reply:
Changed.